### PR TITLE
Updated for FW v5 compatibility

### LINKED ISSUE #5: needs updating for FW v5 compatibility

### PR DESCRIPTION
**Pull request by moloko**
Fixes #5

### CODE REVIEW

**Comment by moloko** [comment]
@guywillis if you have a spare moment could you sort out the  button positioning?

File: .bowerrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (oliverfoster) [inline .bowerrc]: Remove file?

File: bower.json [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-    ],
-    "license": "GPL-3.0",
-    "targetAttribute": "_submitAll"
+  "name": "adapt-submitAll",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/cgkineo/adapt-submitAll.git"
+  },
+  "version": "3.0.0",
+  "framework": ">=5.3",
```
Comment (tomgreenfield) [inline bower.json:8]: Needs to be `>=5.8` due to ES9 syntax.

Comment (moloko) [reply bower.json:8]: what ES9 syntax?

Comment (moloko) [reply bower.json:8]: Oh, is it `...saData` on L138? I didn't realise that specific form of spread syntax was ES9

**Pull request merged**